guacamole/src/main/frontend/src/app/settings/services/preferenceService.js

```diff
@@ -142,6 +142,20 @@ angular.module('settings').provider('preferenceService', ['$injector',
          */
         language : getDefaultLanguageKey(),
         
+        /**
+         * The number of recent connections to display.
+         * 
+         * @type Integer
```

Comment:
This should be `@type {number}` (or `@type {!number}` if not nullable).

Reply:
Fixed via rebase.